### ISSUE TITLE
Catalog view, column "addon" does not resize

### ISSUE THREAD

**Issue by Bykow** [casperstorm/ajour#361]
**Describe the bug**
When browsing the catalog, one can't resize the column in which the name of the addon appear. It, instead, resizes the "description" column

**To Reproduce**
1. Catalog
2. Drag the right edge of column to resize
3. bug

**Expected behavior**
Resizes the correct column

**Screenshots**
Nop

**Software involved**
Tested on Win10, Ajour 0.5.1

**Comment by tarkah** [comment]
@Bykow can you share a screenshot of the order you have your columns in? I cannot recreate this issue on 0.5.1 with windows 10.

**Comment by casperstorm** [comment]
Please see https://github.com/casperstorm/ajour/issues/353.

@tarkah i think it must be an error if a user has a modified value and then we did some core changes to the columns which is the issue.

**Comment by ascott18** [comment]
![image](https://user-images.githubusercontent.com/5017521/100526150-07691e00-317b-11eb-81aa-5378d481e6c6.png)

Here's a reproduction of this. Can't see the full names of the addons, attempting to resize the Addon column resizes the Description column instead.

Column config (I've customized My Addons columns, but not Catalog columns):
![image](https://user-images.githubusercontent.com/5017521/100526165-2071cf00-317b-11eb-84c8-c11d8da0864e.png)

**Comment by SimonVT** [comment]
I experienced the same issue, where the Addon column gets a fixed width.
Started from a fresh install, but I'm not able to retrace the steps that led to the issue.

Attached the yml that has the issue:
[ajouryml.txt](https://github.com/casperstorm/ajour/files/5612383/ajouryml.txt)

**Comment by tarkah** [comment]
If you're experiencing this issue, make sure to delete the `column_configs` section from the config file `ajour.yml` then reboot. It appears there is some unknown bug that happens when upgrading versions that causes this to get messed up.

**Comment by ascott18** [comment]
My messed up column config:

```
column_config:
  V3:
    my_addons_columns:
      - key: status
        width: 85
        hidden: false
      - key: title
        width: ~
        hidden: false
      - key: local
        width: 150
        hidden: false
      - key: remote
        width: 150
        hidden: false
      - key: channel
        width: 85
        hidden: false
      - key: author
        width: 85
        hidden: true
      - key: game_version
        width: 110
        hidden: true
      - key: date_released
        width: 110
        hidden: true
      - key: source
        width: 110
        hidden: true
    catalog_columns:
      - key: install
        width: 99
        hidden: false
      - key: addon
        width: 150
        hidden: false
      - key: description
        width: 500
        hidden: false
      - key: source
        width: 89
        hidden: false
      - key: num_downloads
        width: 105
        hidden: false
      - key: game_version
        width: 105
        hidden: true
      - key: date_released
        width: 105
        hidden: false
```

Note the absence of a `~` width on any of the catalog columns. I manually edited my messed up config and changed the `addon` catalog column width to `~` and it works fine now.

**Comment by tarkah** [comment]
Yeah that's exactly what happened. The ~ indicates the column that we use as fill width, to take up the remaining space. I believe quite a few versions ago we switched which column was the fill column and it must have not saved properly to the config in certain instances.

**Comment by tarkah** [comment]
A note for myself to fix before next release... We should add some logic to always force this column to fill width so this error doesn't occur, even if the config gets messed up.

**Comment by tarkah** [comment]
I've applied the fix in #420. This width for Title in the config is effectively ignored and always set to `Length::Fill`.
